### PR TITLE
🏗 Update `@ampproject/filesize` to 4.0.1 with new API

### PR DESCRIPTION
**Pull request by kristoferbaxter**
Filesize 4.0.1 was released with improved concurrency, the ability to filter types of compression, and an updated API.

These changes have been applied in this PR.

|             | Local  | Travis |
|---        |---       |---       |
| Before | 4.23s  | 16s     |
| After    | **2.55s**  | **7.74s**  |

### CODE REVIEW

**Comment by kristoferbaxter** [comment]
I restarted the bundle size check and it worked successfully.

File: package.json [hunk 1]
```diff
@@ -196,6 +196,7 @@
       "dist/v0/*-?.?.js",
       "dist/*.mjs",
       "dist/v0/*-?.?.mjs"
-    ]
+    ],
+    "trackFormat": ["brotli"]
```
Comment (erwinmombay) [inline package.json:200]: this filters out the reporting to just brotli correct?

Comment (kristoferbaxter) [reply package.json:200]: This option specifies which formats (compression formats) to track. In the case of AMP, the bundlesize check only cares about `brotli`.

Comment (rsimha) [reply package.json:200]: I just noticed that `yarn add` and `prettier` [do not agree](https://travis-ci.org/github/ampproject/amphtml/jobs/678816324#L406-L411) on how this setting should be formatted inside `package.json`. I've filed https://github.com/ampproject/filesize/issues/83 to request that the suggestion in https://github.com/ampproject/amphtml/pull/26907#discussion_r383399093 (a separate config file for `filesize`) be implemented.

**Pull request merged**